app/poros/nytimes_results.rb

```diff
@@ -30,6 +30,23 @@ def get_obituaries_by_date(beginning_date, ending_date)
     end
   end
 
+  def get_most_recent_obituaries
+    json = NytimesService.new.get_most_recent_obituaries
+    json[:response][:docs].map do |obituary_data|
+      Obituary.new(format_obituary_data(obituary_data))
+    end
+  end
+
+  def get_advanced_obituaries(name, date)
```

Comment:
Using the name and date as params here, I will have it ignore the state, age, and city params if they use the NYT API

Reply:
That makes sense. I guess we will have to figure out a way to let users know on the advanced search page that state, age and city will only search our community obits.

Reply:
I imaged have a little line between the search fields to have a slight sectioning off.